src/fallback.ts

```diff
@@ -421,15 +420,19 @@ export function applyTryTacticToBlock(
       declarations[key] ??= 'revert';
     }
 
-    // todo: This does not support anchor functions that are nested or passed
-    // through custom properties.
-    if (isAnchorFunction(valueAst.children.first)) {
-      valueAst.children.first.children.forEach((item) => {
-        if (isIdentifier(item) && isAnchorSide(item.name)) {
-          item.name = mapAnchorSide(item.name, tactic);
+    // todo: This does not support anchor functions that are passed through custom properties.
+    csstree.walk(valueAst, {
+      visit: 'Function',
+      enter(node) {
+        if (node.name === 'anchor') {
```

Comment:
```suggestion
        if (isAnchorFunction(node)) {
```
While `isAnchorFunction` will recheck that it's a function despite the fact that we already know it's a function based on `visit: 'Function'`, I'd prefer to keep using the shared util.